src/Tribe/Views/V2/Hooks.php

```diff
@@ -713,4 +715,28 @@ public function print_single_json_ld() {
 
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
+	 * Add views stylesheets to customizer styles array to check.
+	 * Remove unused legacy stylesheets.
+	 *
+	 * @param array $sheets Array of sheets to search for.
+	 *
+	 * @return array Modified array of sheets to search for.
+	 */
+	public function customizer_inline_stylesheets( $sheets, $css_template ) {
+		$v2_sheets = [
+			'tribe-events-views-v2-skeleton',
+			'tribe-events-views-v2-full',
+		];
+
+		// Unenqueue legacy sheets.
+		if ( ( $keys = array_keys( $sheets, 'tribe-events-calendar-style' ) ) !== [] ) {
+			foreach( $keys as $key ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after FOREACH keyword; 0 found
* :no_entry_sign: There must be a space between foreach and the first parenthesis '('

<sub>hash: fe7a821778db6a04a5a8e8afaf9140e4</sub>